### DISCUSSION TITLE
Returning the Parsed Data & cURL parameters

### DISCUSSION THREAD

**Discussion by baruwaetti** [fureweb-com/public-google-sheets-parser#18] [Q&A] [answered]
Great library and thanks a lot for sharing!

Quick question,
How do return the data (items) in a Node environment?
Also for the cURL, does it accept the sheet name as a parameter as well?

Thanks again.

**Comment by SoulAuctioneer** [comment, marked as answer]
You can see examples for passing a sheet name in the README.md

Returning data in a Node environment should be no different from doing so in the browser. Just follow the examples in the README.md.

**Comment by fureweb-com** [comment]
@baruwaetti Please check below section of documentation :)

https://github.com/fureweb-com/public-google-sheets-parser/blob/main/README.md#usage-example

If you still have a problem, you can give more information to solve your problem.
